OpenOversight/tests/test_utils.py

```diff
@@ -19,13 +22,24 @@
 
 
 # Utils tests
-
 upload_s3_patch = patch(
     "OpenOversight.app.utils.cloud.upload_obj_to_s3",
     MagicMock(return_value="https://s3-some-bucket/someaddress.jpg"),
 )
 
 
+class PoliceDepartment:
```

Comment:
What will this be used for? It seems to be basically the same as the class in database.py -  all the same attributes

Reply:
It's used an easier and more standard alternative to code like this:
```python
name = "Added Police Department"
short_name = "APD"
unique_internal_identifier = "30ad0au239eas939asdj"
```